src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptInProcLanguageClient.cs

```diff
@@ -57,6 +58,21 @@ public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapa
         serverCapabilities.ProjectContextProvider = true;
 
         serverCapabilities.SupportsDiagnosticRequests = true;
+        serverCapabilities.DiagnosticProvider = new()
+        {
+            SupportsMultipleContextsDiagnostics = true,
+            DiagnosticKinds =
+            [
+                new(PullDiagnosticCategories.Task),
+                new(PullDiagnosticCategories.EditAndContinue),
```

Comment:
my guess is that this one is very not necessary, and possibly bad (since it will pull in enc).